lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -81,6 +81,7 @@ class InstructorDashboardTab(CourseTab):
     title = ugettext_noop('Instructor')
     view_name = "instructor_dashboard"
     is_dynamic = True    # The "Instructor" tab is instead dynamically added when it is enabled
+    priority = 300
```

Comment:
A higher number here gives broader scope for external course tab plugins to fit in somewhere in between.